src/content/docs/en/guides/integrations-guide/markdoc.mdx

```diff
@@ -283,45 +288,54 @@ export default defineMarkdocConfig({
 
 <ReadMore>To learn about configuring Prism stylesheets, [see our syntax highlighting guide](/en/guides/markdown-content/#prism-configuration).</ReadMore>
 
-### Set the root HTML element
+<ReadMore>See the [Markdoc nodes documentation](https://markdoc.dev/docs/nodes#built-in-nodes) to learn about all the built-in nodes and attributes.</ReadMore>
```

Comment:
Does this belong here? It looks like it's part of the current Syntax Highlighting section when it's here. What do you think about moving this to line 312, right after the code sample about standard nodes/elements?